frontend/src/app/modules/fields/display/field-types/wp-spent-time-display-field.module.ts

```diff
@@ -65,5 +70,35 @@ export class WorkPackageSpentTimeDisplayField extends DurationDisplayField {
 
     element.innerHTML = '';
     element.appendChild(link);
+
+    this.appendTimelogLink(element);
+  }
+
+  private appendTimelogLink(element:HTMLElement) {
+    this
```

Comment:
This method will result in 1 request per work package in the list and we should work on that before merging.

Looks like there is `workPackage.logTime` that should do the same thing?

Reply:
Thanks! 👍 I missed that link.